src/api_server/Cargo.toml

```diff
@@ -41,6 +39,15 @@ tokio.workspace = true
 tonic = { version = "0.8", optional = true }
 uuid = { version = "1.2.2", features = ["serde", "v4"] }
 
+[target.'cfg(not(target_arch = "s390x"))'.dependencies]
```

Comment:
I think it would be more generic to define the openssl vs rustls independently from the architecture. If someone wants to build kbs with `openssl` they should build with a `--features openssl` option instead. The default should be rustls imho, but openssl would be a build time option that is not architecture dependent.

@Xynnn007 @jialez0 WDYT?

Reply:
Yes. I agree. As `rustls`-`ring` does not work on `s390x`, and `openssl` can. It will be helpful for folks working on that arch. 

We need to impl this from the AS side.

Reply:
@sameo @Xynnn007 @jialez0 I see. Let me add two features. Are the following feature names suitable?

- rustls
- openssl

Reply:
It is perfect to me!